pkgs/development/python-modules/django-extensions/default.nix

```diff
@@ -0,0 +1,25 @@
+{
+  stdenv, buildPythonPackage, fetchFromGitHub,
+  vobject, mock, tox, pytestcov, pytest-django, pytest, shortuuid, django
+}:
+
+buildPythonPackage rec {
+  pname = "django-extensions";
+  version = "1.8.1";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "${pname}";
+    repo = "${pname}";
+    rev = "${version}";
+    sha256 = "08rd9zswvjb9dixzyd3p3l3hw3wwhqkgyjvid65niybzjl1xdb5h";
+  };
+
+  buildInputs = [ django vobject mock tox pytestcov pytest-django pytest shortuuid];
```

Comment:
Thank a lot. I don't really understand build system (`buildInputs`, `propagatedBuildInputs`, `nativeBuildInputs`, etc). I try reading [manual](https://nixos.org/nixpkgs/manual/#handling-dependencies) but doesn't fully get them.

Never have any problem with current expression, so I assume I wrote it correctly.